### PR TITLE
Fix branch name in GL CI

### PR DESCRIPTION
**Pull request by mateuszrzeszutek**

### CODE REVIEW

File: .gitlab-ci.yml [hunk 1]
```diff
@@ -20,7 +20,7 @@ build:
 sast-scan:
   stage: verify
   rules:
-    - if: '$CI_COMMIT_REF_NAME == "main"'
+    - if: '$CI_COMMIT_REF_NAME == "master"'
```
Comment (jkwatson) [inline .gitlab-ci.yml:23]: wait...why is this going *to* master from main? Shouldn't we be using main everywhere?

Comment (mateuszrzeszutek) [reply .gitlab-ci.yml:23]: We haven't done the master->main migration in all repos, only those that are going to be supported long term I guess?

Comment (mateuszrzeszutek) [reply .gitlab-ci.yml:23]: Although we're probably going to support this repo for a while; it'd make sense to do the migration.

**Review by jkwatson — APPROVED** [review]

**Comment by breedx-splk** [comment]
Tracked it with #160 .

**Pull request merged**